packages/webdriverio/src/scripts/isElementClickable.js

```diff
@@ -4,7 +4,19 @@
  * @return {Boolean}           false if element is not overlapped
  */
 export default function isElementClickable (elem) {
-    if (!elem.getBoundingClientRect || !elem.scrollIntoView || !elem.contains || !elem.getClientRects || !document.elementFromPoint) {
+    if (!elem.getBoundingClientRect || !elem.scrollIntoView || !elem.getClientRects || !document.elementFromPoint) {
+        return false
+    }
+
+    // check if the childElement is contained in containerElement with ShadowDom support
```

Comment:
@jimmy-collazos the solution you've proposed works great for Edge because it doesn't support shadow dom, polyfills are used instead like shadydom.
I'd like to use it to fix https://github.com/webdriverio/webdriverio/issues/4804